app/controllers/category/index.js

```diff
@@ -11,9 +11,6 @@ export default Ember.Controller.extend(PaginationMixin, {
 
     totalItems: computed.readOnly('model.meta.total'),
 
-    categoryController: Ember.inject.controller('category'),
-    category: computed.alias('categoryController.model'),
-
```

Comment:
@carols10cents from what I can tell removing these lines will break https://github.com/rust-lang/crates.io/blob/master/app/templates/category/index.hbs because `category` is no longer available